api/server.go

```diff
@@ -42,6 +44,10 @@ const (
 	GPU_CONTROLLER_LOG_PATH = "/tmp/cedana-gpucontroller.log"
 )
 
+const (
+	port = 9999
+)
```

Comment:
Can rename this variable as `PORT` or `VSOCK_PORT` as all our constants are in uppercase.